Functions/CatalogItems/Remove-RsCatalogItem.ps1

```diff
@@ -54,4 +72,5 @@ function Remove-RsCatalogItem
         Write-Error "Exception occurred while deleting catalog item! $($_.Exception.Message)"
         break
     }
+    }
```

Comment:
Please correct the indentation (the content should be one level deeper than the bracket